python/dataproc_templates/jdbc/jdbc_to_gcs.py

```diff
@@ -148,10 +154,10 @@ def parse_args(args: Optional[Sequence[str]] = None) -> Dict[str, Any]:
         known_args: argparse.Namespace
         known_args, _ = parser.parse_known_args(args)
 
-        if getattr(known_args, constants.JDBCTOGCS_INPUT_TABLE) and getattr(known_args, constants.JDBCTOGCS_SQL_QUERY):
+        if getattr(known_args, constants.JDBCTOGCS_INPUT_TABLE) and getattr(known_args, constants.JDBCTOGCS_INPUT_SQL_QUERY):
             sys.exit('ArgumentParser Error: Arguments cannot have both input table and sql query, use either one.')
 
-        if getattr(known_args, constants.JDBCTOGCS_SQL_QUERY) and not getattr(known_args, constants.JDBCTOGCS_TEMP_VIEW_NAME):
+        if getattr(known_args, constants.JDBCTOGCS_TEMP_SQL_QUERY) and not getattr(known_args, constants.JDBCTOGCS_TEMP_VIEW_NAME):
```

Comment:
What will happen if temp view is given but temp query is empty?

Reply:
The temp view will be created only if temp query is given. `if temp_sql_query : ` -> create temp view & execute query